### PR TITLE
Signin and cart completed

### PR DESCRIPTION
**Pull request by sakshamAK**
Added the sign in page
Added sign up page
Added cart page with total amount and items details

image
image
image
image

### CODE REVIEW

**Review by gopalbbharadva — APPROVED** [review]
Looks good:)

File: mycart/style.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
+@import url("../../css/color.css");
+
+body {
+    font-family: Roboto, sans-serif;
```
Comment (srishti-maurya) [inline mycart/style.css]: I guess you can set the `font-family` in any one css file rather than in all

**Review by srishti-maurya — APPROVED** [review]
UI looks good!
You may have a common stylesheet too for the repeated styles

File: mycart/style.css (hunk 1 above)
Comment (sakshamAK) [reply mycart/style.css]: Sure, thanks

**Pull request merged**